packages/react-slider/src/components/Slider/stories/SliderControlled.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { Label } from '@fluentui/react-label';
+import { Slider } from '../../../index'; // codesandbox-dependency: @fluentui/react-slider ^9.0.0-beta
+
+export const Controlled = () => {
+  const [sliderValue, setSliderValue] = React.useState(160);
+  const onSliderChange = (
+    ev: React.PointerEvent<HTMLDivElement> | React.KeyboardEvent<HTMLDivElement>,
+    data: { value: number },
+  ) => setSliderValue(data.value);
+
+  return (
+    <>
+      <Label htmlFor={'controlled-slider'}>Controlled Example [ Current Value: {sliderValue} ]</Label>
```

Comment:
```suggestion
      <Label htmlFor='controlled-slider'>Controlled Example [ Current Value: {sliderValue} ]</Label>
```

Reply:
```suggestion
      <Label htmlFor="controlled-slider">Controlled Example [ Current Value: {sliderValue} ]</Label>
```

@layershifter  your suggestion should use double quotes, single quotes will just error